packages/playground/testUtils.ts

```diff
@@ -66,8 +66,12 @@ export function readFile(filename: string) {
   return fs.readFileSync(path.resolve(testDir, filename), 'utf-8')
 }
 
-export function editFile(filename: string, replacer: (str: string) => string) {
-  if (isBuild) return
+export function editFile(
+  filename: string,
+  replacer: (str: string) => string,
+  runInBuild: boolean = false
+) {
```

Comment:
```suggestion
): void {
```

(_Our linter doesn't check playgrounds_ :thinking:)